### ISSUE TITLE
training time

### ISSUE THREAD

**Issue by ZhongshanDu** [CNJianLiu/Diff9D#9]
Hello, I tried your training setup, and it takes around 15 minutes per epoch on my device. How long did it take to complete 1000 epochs when training on the NVIDIA RTX 3090 GPU, as mentioned in your paper?

**Comment by CNJianLiu** [comment]
Hi, ~10 days on a single 3090 GPU, but you can try to evaluate it after 200 epochs (~2 days), the accuracy is near the final

**Comment by ZhongshanDu** [comment]
thank you！！！
